application/libraries/MY_Form_validation.php

```diff
@@ -38,11 +38,11 @@ public function has_error( $field = NULL )
          *
          * @return bool Success or Failure
          */
-//    public function run($module = '', $group = '')
-//    {
-//        is_object($module) && $this->CI =& $module;
-//        return parent::run($group);
-//    }
+        public function run($module = '', $group = '')
```

Comment:
You're right. I'm good for switching the placement of those around. Will need to modify CIDbModel also for when it calls that. I think that's the only place it would affect.